src/sql/workbench/services/objectExplorer/browser/objectExplorerViewTreeShim.ts

```diff
@@ -131,8 +133,12 @@ export class OEShimService extends Disposable implements IOEShimService {
 		return this.sessionMap.get(key)!;
 	}
 
-	public async getChildren(node: ITreeItem, viewId: string): Promise<ITreeItem[]> {
+	public async getChildren(node: ITreeItem, viewId: string, configurationService: IConfigurationService): Promise<ITreeItem[]> {
 		if (node.payload) {
+			if (node.payload.authenticationType !== undefined && node.payload.authenticationType === '') {
```

Comment:
Bugfix? In JS, the truthiness of empty string is false.  So, if the value of `node.payload.authenticationType` is set to empty string, it can't be both true and equal to empty string.  Updated the if statement to what I think the author was trying to do.... If the payload object has the `authenticationType` _property_ and it's set to an empty string, then set defaults.